cfme/tests/automate/custom_button/__init__.py

```diff
@@ -117,6 +117,20 @@ def is_displayed(self):
         return self.submit.is_displayed and self.service_name.is_displayed
 
 
+class CredsHostsDialogView(View):
+    """This view for custom button default ansible playbook dialog"""
+
+    machine_credential = BootstrapSelect(locator=".//select[@id='credential']/..")
```

Comment:
I would recommend changing the xpath expression here to avoid use of `/..`. The element type selection should be more specific - is it a div? `.//div[.//select[@id='credential']]`

Reply:
I moved it like `".//select[@id='credential']//parent::div"`